pulp_rpm/app/tasks/copy.py

```diff
@@ -57,7 +57,7 @@ def find_children_of_content(content, src_repo_version):
         children.update(packages.filter(advisory_package_q).values_list("pk", flat=True))
 
         module_nsvcas = advisory.get_module_list()
-        advisory_module_q = Q()
+        advisory_module_q = Q(pk__in=[])
```

Comment:
Empty ```Q()``` object matches everything, not nothing. Therefore if ```advisory.get_module_list()``` doesn't return any modules but there are modules in the repo, they all get copied.